ipatests/pytest_ipa/integration/tasks.py

```diff
@@ -1969,3 +1969,9 @@ def remote_ini_file(host, filename):
 def is_selinux_enabled(host):
     res = host.run_command('selinuxenabled', ok_returncode=(0, 1))
     return res.returncode == 0
+
+
+def get_sssd_version(host):
+    """Get sssd version on remote host as a tuple of integers."""
+    version = host.run_command('sssd --version').stdout_text.strip().split('.')
+    return tuple(map(int, version))
```

Comment:
You may use the `pkg_resources.parse_version` method that returns a Version.

Reply:
This will require importing parse_version from pkg_resources in every module where we need to  check the version. 
Also I hope that this approach will help to avoid invalid comparisons like
```
if sssd_version.version < '2.2.0':
```
which I suppose appeared in the code because usage of Version object is not obvious.

Reply:
This code is working but IMO fragile if the version ever contains a char like 2.2.1b or an epoch tag 1:2.2.2